src/App/App.js

```diff
@@ -28,8 +29,34 @@ class App extends Component {
           userDeckNames.push(deck.deckName)
         }
       })
-    })
-    this.setState({ userDecks: newUserDecks });
+    });
+    this.setDeckPrices(newUserDecks);
+  }
+
+  setDeckPrices(userDecks) {
+    const newUserDecks = userDecks.map(deck => {
+      deck.price = 0;
+      Object.keys(deck.cardCounts).forEach(cardInDeck => {
+        const cardPrice = this.state.cards.find(card => {
+          return card.cardName === cardInDeck;
+        }).price;
+        const userCard = this.state.userCardsData.find(card => {
```

Comment:
What is .price doing here? I have never seen that done. 😄

Reply:
The find method returns an object from this.state.cards, so .price is getting the value of the price property from that object.